app/controllers/admin/problems_controller.rb

```diff
@@ -1,4 +1,74 @@
 # frozen_string_literal: true
 
 class Admin::ProblemsController < Admin::ApplicationController
+  before_action :set_session
+  before_action :set_mission
+  before_action :set_problem, only: %i[show]
+
+  def new
+    @problem = @mission.problems.new
+  end
+
+  def create
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for create is too high. [48.75/15]